src/examples/FileInputForm.js

```diff
@@ -0,0 +1,37 @@
+import React, {Component, PropTypes} from 'react';
+import {reduxForm} from 'redux-form';
+export const fields = ['avatar'];
+
+class FileInputForm extends Component {
+  static propTypes = {
+    fields: PropTypes.object.isRequired,
+    handleSubmit: PropTypes.func.isRequired,
+    resetForm: PropTypes.func.isRequired
+  };
+
+  render() {
+    const {
+      fields: {avatar},
+      handleSubmit,
+      resetForm
+      } = this.props;
+    return (<form className="form-horizontal" onSubmit={handleSubmit}>
+        <div className="form-group">
+          <label className="col-xs-4 control-label">Avatar</label>
+          <div className="col-xs-8">
+            <input type="file" className="form-control" {...avatar} value={ null } />
```

Comment:
Perhaps it would be better to demonstrate the alternative which is pulling `value` out of `avatar` and then spreading the rest of the properties?